adsbibdesk.py

```diff
@@ -689,6 +680,8 @@ def get_redirect(url):
         red_url=out.geturl()
     except URLError as out:
         red_url=out.geturl()
+    #except socket.timeout as e:
+    #    red_url=url
```

Comment:
Was this an experiment?  Perhaps remove these commented-out lines unless there's a reason to keep them.

Reply:
okay, I am getting rid of it. I had some strange problems when using the off-campus network last week, and this testing line was left here accidentally.